test/python/test_hybridmap.py

```diff
@@ -0,0 +1,94 @@
+"""Test the hybrid Neutral Atom mapper."""
+
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+import pytest
+from qiskit import QuantumCircuit
+
+from mqt.qmap import HybridMapperParameters, HybridNAMapper, NeutralAtomHybridArchitecture
+
+json_data = {
+    "name": "rubidium",
+    "properties": {
+        "nRows": 5,
+        "nColumns": 6,
+        "nAods": 1,
+        "nAodCoordinates": 5,
+        "interQubitDistance": 3,
+        "minimalAodDistance": 0.1,
+        "interactionRadius": 3,
+        "blockingFactor": 1,
+    },
+    "parameters": {
+        "nQubits": 20,
+        "gateTimes": {
+            "none": 0.5,
+            "rx": 0.5,
+            "rz": 0.5,
+            "h": 0.5,
+            "x": 0.5,
+            "p": 0.001,
+            "cx": 0.7,
+            "cz": 0.2,
+            "ccz": 0.4,
+            "cccz": 0.6,
+            "ccccz": 0.8,
+            "mcx": 2.5,
+        },
+        "gateAverageFidelities": {
+            "none": 0.999,
+            "h": 0.999,
+            "x": 0.999,
+            "rx": 0.999,
+            "rz": 0.999,
+            "p": 0.999,
+            "cz": 0.995,
+            "ccz": 0.95,
+            "cccz": 0.95,
+            "ccccz": 0.95,
+            "cccccz": 0.95,
+            "mcx": 0.95,
+        },
+        "decoherenceTimes": {"t1": 100000000, "t2": 1500000},
+        "shuttlingTimes": {"move": 0.55, "aod_move": 0.55, "aod_activate": 20, "aod_deactivate": 20},
+        "shuttlingAverageFidelities": {"move": 1, "aod_move": 1, "aod_activate": 1, "aod_deactivate": 1},
+    },
+}
```

Comment:
I actually had the parametrized tests and used the files based on __file__. But I couldn't make it run on the CI (maybe you can check some commits in the past). I also tested the creation of temporary files but also couldn't really make it work, so this is the workaround which works the best.

Reply:
I think you were on the right track in that commit where you were using `__file__`. You just didn't get the path quite right I believe. IIRC, it should be `Path(__file__).parent.parent` to get out of the `python` and into the `test` folder. Maybe worth giving it one more try

Reply:
Maybe, but the way I had it, it worked on my local machine. So even if it would work with the changed path in the CI, it would probably break my local version which is also non-optimal.

Reply:
Getting the paths right is always tricky. In the end, this kind of duplication is not the end of the world. So I'd be ok with just leaving it the way it is. However, there is definitely a way to fix this that only requires moderate effort and is expected to work locally and in CI.

Reply:
There should be a solution I guess. But debugging the CI is almost impossible (or at least I don't know how).

Reply:
I'll add this to the list of things that I'll try to check once most other things are resolved. Then I can cover the C++ tests and files together with the Python tests. I've done my fair share of fiddling around with CI and the tests in the past. So, ideally, this will be a rather quick fix.